### PR TITLE
#7 Task. Elvin Zeinalabdziyeu

### PR DESCRIPTION
**Pull request by gurman-eez**
https://travis-ci.org/github/gurman-eez/js-assignments/builds/718156812

### CODE REVIEW

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -46,8 +59,27 @@ function* get99BottlesOfBeer() {
...
  */
+
+
 function* getFibonacciSequence() {
-    throw new Error('Not implemented');
+	//  throw new Error('Not implemented');
+	let x = 0,
+		 y = 1;
+	while (x >= 0) {
+		if (x == 0) {
```
Comment (vramaniuk) [inline task/07-yield-tasks.js]: "operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"

**Review by vramaniuk — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
> https://travis-ci.org/github/gurman-eez/js-assignments

Please, privide the direct link to your task success travis build
And resolve conflicting files (readme.md)

File: task/07-yield-tasks.js (hunk 1 above)
Comment (gurman-eez) [reply task/07-yield-tasks.js]: fixed

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**